dev-docs/source/images/architecture-container.drawio

```diff
@@ -0,0 +1 @@
X+LmbmrvpxDD0SJsHXtAAf7v6+Xj7zyLsBT+Gywl57CnxKK/ACfmC+WTJJqEARqE3hWwQ+UwdrhY2gWMfWOzuivKcti2I6/DbAcFoCS+Rg3D0tCpJ8tXwkt6ZIY/k2mfRh7XbjpO0e8ijww6nIFikLwyWkFiMkhK9mDsgCPj36vo5SV4hJnCda+L0uIXIhQRvaBd+V+Os4bKpJrK5yjht6pyfixyXNZV3BFy65unQGQPoF84DMT++DNyv9xv98udI/t+rpW2C0dLvadv5MacM8d+0/lQ7wCQZRGqkSyrenDCygDCyoVQJI2v6/oRxbm5fg1+36uzrw/3LD+vuq75+6Sn6dsrQYSgswO1SCgI/xoqZvWaCNsxkXNpbiKP+qnmhCnnUyPXWgqsr5wPJ1GVN06W+YUp6hT26dq4Zui6biiar5kAVMKumS/esEwm14dBZDCf0y5x9uV77ENsuJRNg4z0FECd96DvTbulzOG3Rhw8QB8g7068Ed2tbGCcL0mO8hCi50QsiO3RBO8iSv85uJqPE85PQjNGLmRVg2Y5NNhEPKAuoxgGfhJiKFDV9gER9MHLpHw+GVLw8RsmQ/ZFguvQgt+R4fvHbzo+NyQBb3BJToTiidG9SwRwUPu3Apz8QPraPPAtt5uA0MHpQNF66pJxXYeBwKC0kTeK1fXSUbuR7a5Q2pNNEaTHrZAHrSih9ERL0HU5Di9gMulri8xjNyIoSlw4+3gQEum+E6sAH3u5QzeZMfXGbuvQO4w1m86cRRYrLICjC9YT1sj0qXmEZlOOZJLP7IJjcjSz3i2BzMgj98Gz882M2uoN/fLu3Qk8S9sJevSjXypIV05LJEZ5P/lJ0qqwUzOj7pdL3v6syNrJfmXRfxQI1JhgCt1ZsMjHvehqHXeQWTW6vHtJ29ShJ9zD6103MZyQBepKQGAhimyS+ywuv0UHMd/XsX/8hCukR1x77o8nL73CS+Ee1tEKYLNAcecAZIeRzCv2GhGw4IAAKcUX6FSJouLbJTzYW9Q/iq1+5O1dr/proYpNceHRluYfY5a/8veyx6Cp5rpZBAQqxxZc3vF+6L/+VV4GJnsCvsbMaSctEZwnAc0ga+nFYgdNC3qbKbgwdivqvxTSOiHf80QdkRxEBFxOKqAUxMaQS++OJ8qdKEpBOY3ehUNQKhtGgI6hayb8eNlQ8vL+rfhal0AhMoFPyqhx77tHvFuUQDWLUIVMdZh0v+A3Xnk7ZGEMMqW3l7ijjrc/WGq1eHzLFj20Vl0FZEXK/SeD394Glc0nVjQKfEvnYlf1JFzSbBfAgnB0IOTXmlzwFtjcC7KrABeTI89fYEUkOjwhq14iwF5onEc9nVNw6r/HNitujmivLp6Sq391HF/+YrMzr69+Lnjkfu1+vEwmsUdVMK6+z1i41t6C3mRq30VyhShxWDbUjaZ2QU1WlGyEwZZHdBamq3vfQI7YL30Pz3qhBRZbVC2mTpsmKqRZtpLyf4pWG6VQPm0Qwx91b6EEcpYofQuyjIAq/YLAkVMkoz31fmIm4pLAHbA/yGCgGYDqjb6RlSuIBo9eoHkn54iBCbG8eh0yBhW2f0QauoRXG6REJeCyfACwLBuwJqug8/RDfvwEUFjaBze5R1KCDRQPzPDGLJKfpioDvO1Ti4gf3STzkK2j1dba6+KuUnLiJPt3EZbJcdLg1qRqXpQW7fFw2OFRcpjZQupO4bEeg3x3KlffyqMQxllGqv+r948ZYcrWm+GlctSR87MJVkw1NKRqQU/LcmiQ9x9lv7MXPNmUYEXC4wTjQCQyj/zuZiSB6XVwp/EZ6q2QCElkAkrcONYZhFnpRK+DlRQu5bmRcImviAi/OVlMxm4EorW0Al0G+NwmyAtTnMxZ6CTnMlsai30HlS4wlVWtxkLD+iLm8xlTA7kZIrRqhpiT+iYT1sllBlMfLhw9kEOQuLYJkJAWhPV2Ao4UQ1XTqDQYuXCG83MkavNkQAGeOsE0WbhBHDLz6yMqMFis9BmnAEIS+T6Ehii0ob6awEDvAxD78gVG4Udg3Ew/6KSFfL8UHxqBaTj8u5OtHQfwPWMrRWoYZynHCDNUsOQvliPHQYYZ0FEH5BK5BW8E5sYx/NT1VE0ZmocTHCyjTrV1d+A+KphU3lOyZkTy8/1Ddw3oZBgS5lJDMZt/lI7CqW1CIyWbUPCrSI7QWnv0SRob/YOHnG5OP/xfBo2a8Y6ZRKFxt9rZ70wt2nIWBACOsbRWpXUcvwUYz9tnXXqT5y51rU1vsQ+Mexc4MRH4rpoDhSdueDsigX3Jd9ZIgxQuvOCCVgfRS2sNUjuzJ9CuC+j3dTsl3vz0FUeDysVKojVrZicHTTe3UTZzR6Ka2qXYXkP09N628DQlbAVLj0ZatgPRuySzhbMwGy94FH/fkx65W5gh8PFZlrLT70FT7xwX66hmXZ0y9PYbz46jU/eGqZE2K0AnEa32lwLNEVk4X8LckJo6C+IkTmnNBM4dU7ISePtpr74X24vxE9VBWnJ+Qbp/uYlGnxHIEGl2XdRammU9Xy7tLXUjnat/cU6/TGvohah9CATAq/O/mFOz2EyH5U6wVgaG8IGcNp6J4YlTwAwHtZUqUoygiWC0ctE8olPLLpqAYne5uKpzDPNTOpWqt8gvwiM0SVRnDTpIbrXPJbzjwUzzvY6pmO+508RsPYueq/nga10crNVuZyiUnAXNNLfRWoXobKWxJdXmwfoGtRXR2rZ3GBgvgs6/WxrEp07C6PQs4idk7mqQNwFrOI6bfh4QOAxNp41zXBTh/wONgWl8riEe/X1Xe5IhJoarYgXSInYw2R6i3JAPfrWYYXT1EFekICvYsJAq2ijRq1FaHK8sAn/e5Rxh9ijKgadq52c8dUzWqItGQLWwf5mlqemY/ET9tx9QgG6phSYMymh06gKz6HFd2sKQtd/en6Tk2K+PunuOemYBesU5WSgPs4jPSy+xHp+Lu2U93qdf/Ag==</diagram></mxfile>
```

Comment:
Github marks the io files not having a new line at the end, I don't think this is critical though.

Reply:
I'd prefer to keep those as saved by drawio as if it's updated then it'll just get overwritten.